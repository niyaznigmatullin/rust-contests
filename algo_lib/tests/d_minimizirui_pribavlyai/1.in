3
1 4 2
9
3 1 3
1 1 3 3
3 1 3
1 1 3 1
3 1 3
2 1 3 5
3 1 3
1 1 3 3
3 1 3
