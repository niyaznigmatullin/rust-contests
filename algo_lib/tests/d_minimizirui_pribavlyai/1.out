7
6
3
18
9
